DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -1530,6 +1572,21 @@ public class DcProvider {
     }
 }
 
+public struct DcReaction: Decodable {
+    let count: Int
+    let emoji: String
+    let isFromSelf: Bool
+}
+
+public struct DcReactions: Decodable {
+    let reactions: [DcReaction]
+    let reactionsByContact: [Int: [String]]
+}
+
+struct DcReactionResult: Decodable {
+    let result: DcReactions
+}
```

Comment:
This is basically mapped from [`reactions.rs`](https://github.com/deltachat/deltachat-core-rust/blob/main/deltachat-jsonrpc/src/api/types/reactions.rs), right?

Reply:
yes.